events.lisp

```diff
@@ -152,29 +150,24 @@
   (dformat 2 "UNMAP: ~s ~s ~a~%" send-event-p (not (xlib:window-equal event-window window)) (find-window window))
   (unless (and (not send-event-p)
                (not (xlib:window-equal event-window window)))
-    (let ((window (find-window window)))
-      ;; if we can't find the window then there's nothing we need to
-      ;; do.
-      (when window
```

Comment:
Please be careful of obliterating comments when you are cleaning up constructions.